podcasts/New Search/Model/SearchResultsModel.swift

```diff
@@ -34,4 +39,42 @@ class SearchResultsModel: ObservableObject {
             episodes = results ?? []
         }
     }
+
+    @MainActor
+    func searchLocally(term searchTerm: String) {
+        let allPodcasts = DataManager.sharedManager.allPodcasts(includeUnsubscribed: false)
```

Comment:
I think we should clear the remote search results when doing another local search since this can cause the remote episodes to stay visible.

```suggestion
        clearSearch()
        let allPodcasts = DataManager.sharedManager.allPodcasts(includeUnsubscribed: false)
```